internal/venice-common/src/main/java/com/linkedin/venice/d2/D2ClientFactory.java

```diff
@@ -41,6 +41,7 @@ public static D2Client getD2Client(String d2ZkHost, Optional<SSLFactory> sslFact
           .setSSLContext(sslFactoryOptional.map(SSLFactory::getSSLContext).orElse(null))
           .setIsSSLEnabled(sslFactoryOptional.isPresent())
           .setSSLParameters(sslFactoryOptional.map(SSLFactory::getSSLParameters).orElse(null))
+          .setRetry(true)
```

Comment:
Based on the javadoc, this `retry` is used to retry the d2 request instead of initialization.
Is it expected?

Reply:
Removed this to see if just increasing the timeout helps